File: examples/02_texture/asset/script/main.lua
-- Create a image
local image = sn.Texture.new()
image:load("logo.png")

function update()
    if sn.Keyboard.isPressed(sn.Keyboard.ESCAPE) then
        sn.Script.load("main", ".")
    end
end

function draw()
    -- Draw image
    sn.Graphics.drawImage(image, sn.Rect.new(sn.Vec2.new(0), image:size()))
end

File: examples/02_texture/asset/script/main.py
# Create a image
import sinen as sn

image = sn.Texture()
image.load("logo.png")


def update():
    if sn.Keyboard.is_pressed(sn.Keyboard.ESCAPE):
        sn.Script.load("main", ".")


def draw():
    # Draw image
    sn.Graphics.draw_image(image, sn.Rect(sn.Vec2(0), image.size()))
